core/esmf-aspect-meta-model-python/esmf_aspect_meta_model_python/vocabulary/constants.py

```diff
@@ -0,0 +1,200 @@
+class MetaModelElements:
+    # Meta Model Elements
+    AbstractEntity = "AbstractEntity"
+    AbstractProperty = "AbstractProperty"
+    Aspect = "Aspect"
+    Boolean = "Boolean"
+    Characteristic = "Characteristic"
+    Code = "Code"
+    Collection = "Collection"
+    Constraint = "Constraint"
+    Duration = "Duration"
+    Either = "Either"
+    EncodingConstraint = "EncodingConstraint"
+    Entity = "Entity"
+    Enumeration = "Enumeration"
+    Event = "Event"
+    FileResource = "FileResource"
+    FixedPointConstraint = "FixedPointConstraint"
+    Language = "Language"
+    LanguageConstraint = "LanguageConstraint"
+    LengthConstraint = "LengthConstraint"
+    List = "List"
+    Locale = "Locale"
+    LocaleConstraint = "LocaleConstraint"
+    Measurement = "Measurement"
+    MimeType = "MimeType"
+    MultiLanguageText = "MultiLanguageText"
+    Operation = "Operation"
+    Point3d = "Point3d"
+    Property = "Property"
+    Quantifiable = "Quantifiable"
+    QuantityKind = "QuantityKind"
+    RangeConstraint = "RangeConstraint"
+    RegularExpressionConstraint = "RegularExpressionConstraint"
+    ResourcePath = "ResourcePath"
+    Set = "Set"
+    SingleEntity = "SingleEntity"
+    SortedSet = "SortedSet"
+    State = "State"
+    StructuredValue = "StructuredValue"
+    Text = "Text"
+    TimeSeries = "TimeSeries"
+    TimeSeriesEntity = "TimeSeriesEntity"
+    Timestamp = "Timestamp"
+    Trait = "Trait"
+    Unit = "Unit"
+    UnitReference = "UnitReference"
+
+    meta_model_elements = (
+        AbstractEntity,
+        Aspect,
+        Characteristic,
+        Constraint,
+        Entity,
+        Event,
+        Operation,
+        Property,
+        Quantifiable,
+        QuantityKind,
+        Trait,
+        Unit,
+    )
+
+
+class CharacteristicElements:
+    # Characteristic Elements
+    Code = "Code"
+    Collection = "Collection"
+    Duration = "Duration"
+    Either = "Either"
+    Encoding_constraint = "EncodingConstraint"
+    Enumeration = "Enumeration"
+    Fixed_point_constraint = "FixedPointConstraint"
+    Language_constraint = "LanguageConstraint"
+    Length_constraint = "LengthConstraint"
+    Locale_constraint = "LocaleConstraint"
+    Measurement = "Measurement"
+    Quantifiable = "Quantifiable"
+    Range_constraint = "RangeConstraint"
+    Regular_expression_constraint = "RegularExpressionConstraint"
+    List = "List"
+    Set = "Set"
+    Single_entity = "SingleEntity"
+    Sorted_set = "SortedSet"
+    State = "State"
+    Structured_value = "StructuredValue"
+    Time_series = "TimeSeries"
+    Trait = "Trait"
+
+    collections = (
+        Collection,
+        List,
+        Set,
+        Sorted_set,
+        Time_series,
+    )
+
+
+class SAMMEElements:
+    TimeSeriesEntity = "TimeSeriesEntity"
+    Point3d = "Point3d"
```

Comment:
I think this should also include [`FileResource`](https://github.com/eclipse-esmf/esmf-semantic-aspect-meta-model/blob/main/esmf-semantic-aspect-meta-model/src/main/resources/samm/entity/2.1.0/FileResource.ttl)